packages/shared/src/hooks/useSquadChecklist.tsx

```diff
@@ -177,12 +177,26 @@ const useSquadChecklist = ({
       actionsPerRoleMap[squad.currentMember?.role] ||
       actionsPerRoleMap[SourceMemberRole.Member];
 
+    const filterPublicStepIfDismissed = (step: ChecklistStepType) => {
+      const isGoPublicDismissed = actions.some(
+        ({ type, completedAt }) =>
+          type === ActionType.HidePublicSquadStep && !!completedAt,
+      );
+
```

Comment:
u can use these func to check if an action is completed 
`  const { checkHasCompleted, isActionsFetched } = useActions();`

because we've had some edge cases it is recommended to check also `isActionsFetched` before doing `checkHasCompleted`

Reply:
Updated! ✅

Reply:
I'm not sure it's updated, I still see this in the PR 
<img width="639" alt="Screenshot 2024-07-25 at 4 51 20 PM" src="https://github.com/user-attachments/assets/0a67dde1-51a0-4dbb-91f9-d66d7d722230">

Reply:
I see u're now using `isActionsFetched` just not `checkHasCompleted`. Placed a suggestion below.